lib/v2/openai/blog.js

```diff
@@ -0,0 +1,89 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const { art } = require('@/utils/render');
+const path = require('path');
+
+module.exports = async (ctx) => {
+    const tag = ctx.params.tag || '';
+
+    const rootUrl = 'https://openai.com';
+    const blogRootUrl = 'https://openai.com/blog';
+    const blogOriginUrl = `${rootUrl}/blog${tag === '' ? '' : `?topics=${tag}`}`;
+
+    // Find API base URL
+    const initResponse = await got({
+        method: 'get',
+        url: blogRootUrl,
+    });
+
+    let apiBaseUrl = initResponse.data
+        .toString()
+        .match(/(?<=TWILL_API_BASE:").+?(?=")/)[0]
+        .replaceAll('\\u002F', '/');
+    apiBaseUrl = apiBaseUrl + '/api/v1/blog-details';
+    const apiUrl = new URL(apiBaseUrl);
+
+    // Construct API query
+    apiUrl.searchParams.append('sort', '-publicationDate,-createdAt');
+    apiUrl.searchParams.append('page[size]', '5');
```

Comment:
```suggestion
    apiUrl.searchParams.append('page[size]', '20');
```

Reply:
> ```suggestion
> 
>     apiUrl.searchParams.append('page[size]', '20');
> 
> ```
> 
> They use 20 by default.

True, I thought it's recommended to reduce the number of entries we get for performance purposes (the list was sliced to 5 previously).